packages/base/src/3dview/mainview.tsx

```diff
@@ -1647,6 +1675,93 @@ export class MainView extends React.Component<IProps, IStates> {
     window.dispatchEvent(resizeEvent);
   }
 
+  private _updateSplit(enabled: boolean) {
+    if (enabled) {
+      if (!this._meshGroup) {
+        return;
+      }
+      this._renderer.setScissorTest(true);
+
+      this._sliderPos = (this._divRef.current?.clientWidth ?? 0) / 2;
+      this._sceneL = new THREE.Scene();
+      this._sceneL.background = SPLITVIEW_BACKGROUND_COLOR;
+      this._sceneL.add(new THREE.AmbientLight(0xffffff, 0.5)); // soft white light
+      const light = new THREE.HemisphereLight(0xffffff, 0x444444, 0.5);
+      this._sceneL.add(light);
```

Comment:
I was wondering why there was a slight delay in the camera movement between the two scenes even though they use the same camera, it seems this delay can be fixed by adding the camera to the sceneL:

```suggestion
      this._sceneL.add(light);
      this._sceneL.add(this._camera);
```